lua/wikis/commons/Infobox/Person.lua

```diff
@@ -446,6 +508,33 @@ function Person:displayLocations()
 	end)
 end
 
+---@param roleKey string
+---@return PersonRoleData?
+function Person._createRoleData(roleKey)
+	if String.isEmpty(roleKey) then return nil end
+
+	local roleData = Roles.All[roleKey:lower()]
+
+	--- Backwards compatibility for old roles
+	if not roleData then
+		local display = String.upperCaseFirst(roleKey)
```

Comment:
Should we also add a tracking category so we can eventually clean the usage of them up?

Reply:
done